src/jit/gentree.cpp

```diff
@@ -505,6 +525,74 @@ bool GenTree::IsNodeProperlySized() const
 
 /*****************************************************************************/
 
+#if MEASURE_NODE_SIZE
+
+void GenTree::DumpNodeSizes(FILE* fp)
+{
+    // Dump the sizes of the various GenTree flavors
+
+#if SMALL_TREE_NODES
+    fprintf(fp, "Small tree node size = %3u bytes\n", TREE_NODE_SZ_SMALL);
+#endif // SMALL_TREE_NODES
+    fprintf(fp, "Large tree node size = %3u bytes\n", TREE_NODE_SZ_LARGE);
+    fprintf(fp, "\n");
+
+#if SMALL_TREE_NODES
+
+    // Verify that node sizes are set kosherly and dump sizes
+    for (unsigned op = GT_NONE+1; op < GT_COUNT; op++)
+    {
+        unsigned    needSize = s_gtTrueSizes[op];
+        unsigned    nodeSize = s_gtNodeSizes[op];
+
+        const char* structNm = OpStructName((genTreeOps)op);
+        const char* operName =       OpName((genTreeOps)op);
+
+        // Have we seen this struct flavor before?
+        for (unsigned mop = GT_NONE+1; mop < op; mop++)
+        {
+            if (strcmp(structNm, OpStructName((genTreeOps)mop)) == 0)
+            {
+                structNm = nullptr;
+                break;
+            }
+        }
+
+        if (structNm != nullptr)
```

Comment:
Let's see - I changed this thing too many times but I think the main thought was that the first time a problem is found the full error message would be output (with the name) and any subsequent errors would still print but without a name. Now that I look at it, though, you're entirely right - it doesn't much matter since this thing better never trigger but might as well simplify the code. Will push in a minute. Thanks.